### PR TITLE
Added anard-overlay

### PR DESCRIPTION
**Pull request by Anard**
Add dpluzz-cli (by melixgaro), an experimental web video downloader, French equivalent of youtube-dl
Maybe a GUI coming soon

### CODE REVIEW

File: files/overlays/repositories.xml [hunk 1]
```diff
@@ -303,6 +303,18 @@
     <feed>https://cgit.gentoo.org/dev/anarchy.git/atom/</feed>
     <!-- <feed>https://cgit.gentoo.org/dev/anarchy.git/rss/</feed> -->
   </repo>
+  <repo quality="experimental" status="unofficial">
+    <name>Anard</name>
```
Comment (mgorny) [inline files/overlays/repositories.xml:307]: Please use a matching case here and in repo_name.

Comment (mgorny) [reply files/overlays/repositories.xml:307]: (as in lowercase/uppercase letters)

**Comment by Anard** [comment]
OK thx, done.
Do I need to do anything to update PR ?
Also ebuild in the repo is not finalized but already functional.

**Comment by mgorny** [comment]
Thanks, it looks good now. I'll squash the commits and merge.

**Pull request closed without merging**